### PR TITLE
Add exploratory data analysis (EDA)

### LINKED ISSUE #9: EDA Analysis for Milestone 1

### PR DESCRIPTION
**Pull request by rtaph**
Addresses #9.

### CODE REVIEW

File: eda/eda_election_turnout.Rmd [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
```
Comment (ChadNeald) [inline eda/eda_election_turnout.Rmd:1]: I needed to create a data/raw folder within the eda folder to knit this file. Is this expected behaviour or should it reference a data/raw folder outside of the eda folder?

File: eda/eda_election_turnout.Rmd [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+## Intro
+
+In this exploratory data analysis (EDA), we will take a first look at our elections data. Our research question of interest is whether more competitive elections are associated with greater voter turnout in British Columbia. To answer this, we will be using open data from Elections BC. Scripts for downloading these data are provided in the `src/` directory.
+
+## Load the data
+
+Let's start by loading the data. We are using two open data sources: the provincial voting results, and the provincial voter participation.  We will these in here and name them `pvr` and `pvp`, respectively.
+
```
Comment (ChadNeald) [inline eda/eda_election_turnout.Rmd:27]: Small typo. "We will use these here and name them..."

File: eda/eda_election_turnout.Rmd [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+f1 <- here::here("data", "raw", "provincial_voting_results.csv")
+pvr <- janitor::clean_names(read_csv(f1))
+
+# Read-in the voter participation data
+f2 <- here::here("data", "raw", "provincial_voter_participation_by_age_group.csv")
+pvp <- janitor::clean_names(read_csv(f2))
+```
+
+Let's take a look a the sample sample or rows from our voter participation dataset by creating an exploratory data table:
+
```
Comment (ChadNeald) [inline eda/eda_election_turnout.Rmd:49]: Small typo: "Let's take a look at a sample of rows from our..."

**Review by ChadNeald — COMMENTED** [review]
You crushed it nicely done! Just a few tiny things to fix and then it looks good to go.

File: eda/eda_election_turnout.Rmd (hunk 1 above)
Comment (rtaph) [reply eda/eda_election_turnout.Rmd:1]: ~~Correct, it requires the two CSVs to live in `data/raw`. I'm assuming that will come in the data-downloading PR?~~

Wait, I misread that. No, it should reference `data/raw` in the root, not within the `eda` folder. `f1 <- here::here("data", "raw", "provincial_voting_results.csv")` should handle that, so I am surprised that it wouldn't have worked. Unless maybe this just has to do with the fact that the data download script isn't on `main` yet, so the `data/raw` doesn't exist yet?

**Comment by rtaph** [comment]
Thanks for the feedback 😃

**Review by ChadNeald — APPROVED** [review]
Nicely done!

**Pull request merged**